### PR TITLE
Improving Customising_prompt.md file 

### PR DESCRIPTION
**Pull request by ayan-joshi**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature , Added a new text file in the prompt named custom_prompt , As mentioned in the issue #471 assigned by @dartpain 

- **Why was this change needed?** (You can also link to an open issue here)
The changes enhance the prompt folder a new text file is explaining the different tasks which GPT can perform and helps the user with what input will provide what output.

- **Other information**:

Please merge my pull request after the check and add the hacktoberfest accepted label to the pull request hanks @dartpain

### CODE REVIEW

**Comment by dartpain** [comment]
Thank you for you contribution.

I dont see how those prompts will be integrated or used within the application at a stage that we are in now. Currently backend requires {summaries} section in the prompt. and a single prompt not multiple examples. 

It seems to me that you want to help people customise prompts. I suggest that you exand this guide
https://docs.docsgpt.co.uk/Guides/Customising-prompts, located here https://github.com/arc53/DocsGPT/blob/main/docs/pages/Guides/Customising-prompts.md

I think this will be a usefule contribution in there

**Comment by ayan-joshi** [comment]
> Thank you for you contribution.
> 
> I dont see how those prompts will be integrated or used within the application at a stage that we are in now. Currently backend requires {summaries} section in the prompt. and a single prompt not multiple examples.
> 
> It seems to me that you want to help people customise prompts. I suggest that you exand this guide https://docs.docsgpt.co.uk/Guides/Customising-prompts, located here https://github.com/arc53/DocsGPT/blob/main/docs/pages/Guides/Customising-prompts.md
> 
> I think this will be a usefule contribution in there

@dartpain That's great so May I start work on this file and commit it to this pull request itself?

**Comment by dartpain** [comment]
Yes please, that exactly what I think will be great!
discard previous commit and just edit https://github.com/arc53/DocsGPT/blob/main/docs/pages/Guides/Customising-prompts.md
I think you can add much more value there.

Thank you!

**Comment by ayan-joshi** [comment]
> Yes please, that exactly what I think will be great! discard previous commit and just edit https://github.com/arc53/DocsGPT/blob/main/docs/pages/Guides/Customising-prompts.md I think you can add much more value there.
> 
> Thank you!

As guided by you I have made the changes @dartpain

**Comment by dartpain** [comment]
Currently the application is using chat_combine_prompt.txt as the main prompt. Please amend. and make sure suggested prompts are based on it:
You are a DocsGPT, friendly and helpful AI assistant by Arc53 that provides help with documents. You give thorough answers with code examples if possible.
Use the following pieces of context to help answer the users question. If its not relevant to the question, provide friendly responses.
You have access to chat history, and can use it to help answer the question.
When using code examples, use the following format:
```(language)
(code)
```
----------------
{summaries}

Thank you

**Comment by ayan-joshi** [comment]
@dartpain have a look and merge if it's okay

File: docs/pages/Guides/Customising-prompts.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+Use the following pieces of context to help answer the users question. If its not relevant to the question, provide friendly responses.
+You have access to chat history, and can use it to help answer the question.
+When using code examples, use the following format:
+
+(code)
+{summaries}
+
+Thank you
+
+
```
Comment (dartpain) [inline docs/pages/Guides/Customising-prompts.md:23]: Can we add one more example
from here https://docs.docsgpt.co.uk/Guides/My-AI-answers-questions-using-external-knowledge

And maybe one check out the https://docs.anthropic.com/claude/docs/let-claude-say-i-dont-know
and https://docs.anthropic.com/claude/docs/ask-claude-to-think-step-by-step
https://docs.anthropic.com/claude/docs/break-tasks-into-subtasks

As a source of inspiration for one or two prompts with some comments next to them

**Review by dartpain — CHANGES_REQUESTED** [review]
Thank you, I left some comments on how we can improve this section

**Comment by ayan-joshi** [comment]
@dartpain if it's ok can you merge it

**Review by dartpain — APPROVED** [review]

**Pull request merged**